ddtrace/internal/core.py

```diff
@@ -102,21 +102,19 @@ def _on_jsonify_context_started_flask(ctx):
 """
 from collections import defaultdict
 from contextlib import contextmanager
+import dataclasses
+import enum
 import logging
 from typing import TYPE_CHECKING  # noqa:F401
 from typing import Any  # noqa:F401
+from typing import Callable  # noqa:F401
+from typing import Dict  # noqa:F401
+from typing import List  # noqa:F401
 from typing import Optional  # noqa:F401
+from typing import Tuple  # noqa:F401
 
 from ddtrace import config
-
-
-if TYPE_CHECKING:  # pragma: no cover
-    from typing import Callable  # noqa:F401
-    from typing import Dict  # noqa:F401
-    from typing import List  # noqa:F401
-    from typing import Tuple  # noqa:F401
-
-    from ddtrace.span import Span  # noqa:F401
+from ddtrace.span import Span  # noqa:F401
```

Comment:
In most of the other places where I've tried to make a similar change (unconditionally importing `Span` for type annotations), it has caused circular import errors. I guess if all of the tests are passing this isn't one of those cases.